portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro

```diff
@@ -430,6 +493,23 @@ definition {
 		Click(locator1 = "DataSet#VIEW_ENTRY_BUTTON");
 	}
 
+	macro saveAction {
+		if (IsElementPresent(locator1 = "RaylifePolicyDeclaration#POLICY_NUMBER", policy = "New Item Action")) {
+			Click(
+				locator1 = "RaylifePolicyDeclaration#POLICY_NUMBER",
+				policy = "New Item Action");
+		}
+		else {
+			Click(
+				key_pageTabTitle = "delete",
+				locator1 = "Testray#PAGE_TAB_TITLE");
+		}
+
+		Click.clickAtNotVisible(
+			key_text = "Save",
+			locator1 = "Button#SAVE");
+	}
```

Comment:
Oh, my bad. I let that go. It was a workaround I did to be able to test it since some parts of Actions were being altered by Juanjo's PR. Now that he has resolved it, it's no longer needed.